host/libhackrf/src/hackrf.h

```diff
@@ -66,6 +66,7 @@ enum hackrf_board_id {
 	BOARD_ID_JELLYBEAN  = 0,
 	BOARD_ID_JAWBREAKER = 1,
 	BOARD_ID_HACKRF_ONE = 2,
+	BOARD_ID_RAD1O = 3,
```

Comment:
I wrote that documentation on the wiki, I wonder why I didn't make sure it matched the code?

Anyway, I've merged this now.  Thank you for your contribution.